veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,185 @@
+# VEP #20: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+This proposal outlines the enhancements implemented in passt and how KubeVirt will integrate with passt to benefit from them:
```

Comment:
Please consider laying out the user-facing changes, e.g the ability to live migrate without loosing TCP connections by leveraging improvements in passt.